CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 *Important!*
 - The next release will be 1.0.0, which means that master will contain breaking changes.
 
+## 1.0.0.pre.2
```

Comment:
We'll probably want to open a PR against master to add this entry in the krane tree too

Reply:
Good shout, will do.